subsys/bluetooth/host/gatt.c

```diff
@@ -1718,13 +1718,9 @@ u8_t bt_gatt_check_perm(struct bt_conn *conn, const struct bt_gatt_attr *attr,
 
 	mask &= attr->perm;
 	if (mask & BT_GATT_PERM_AUTHEN_MASK) {
-#if defined(CONFIG_BT_SMP)
-		if (conn->sec_level < BT_SECURITY_L3) {
+		if (bt_conn_get_security(conn) < BT_SECURITY_L3) {
```

Comment:
There is no IS_ENABLED here. Did you do it differently, commit message doesn't match the diff.

Reply:
It seems to me like the problem is in the commit message. Looking at conn.c `bt_conn_get_security()` evaluates to simply `return BT_SECURITY_L1;` when `CONFIG_BT_SMP=n`, so this doesn't look like it's related to `IS_ENABLED()` in any way.